src/logic/collectibles/sources/OpenSea.ts

```diff
@@ -84,7 +84,10 @@ class OpenSea {
    * @param {string} network
    * @returns {Promise<{ nftAssets: Map<string, NFTAsset>, nftTokens: Array<NFTToken> }>}
    */
-  async fetchAllUserCollectiblesByCategoryAsync(safeAddress, network) {
+  async fetchAllUserCollectiblesByCategoryAsync(
+    safeAddress: string,
+    network: string,
+  ): Promise<{ nftAssets: Map<string, unknown>; nftTokens: Array<unknown> }> {
```

Comment:
Thing is, types were there for the `flow` version, but then were removed after moving to `TS`. That's why JSDOC has those `NFTAsset` and `NFTToken`.

I worked on this same thing in a different PR, I see some conflicts in the near future: https://github.com/gnosis/safe-react/blob/feature/%23934-advanced-settings/src/logic/collectibles/sources/OpenSea.ts

Reply:
Thanks for the tip, I just took your types :)